### DISCUSSION TITLE
[Bug] Feign calls take less time than substeps

### DISCUSSION THREAD

**Discussion by yinbucheng** [apache/skywalking#9356] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

see https://github.com/apache/skywalking/discussions/9353

I'm sure we don't have asynchronous calls anywhere. Kafka sending service is only processed by setting ack=0

![image](https://user-images.githubusercontent.com/30992273/179682632-f9f5dc57-bd7d-4f34-bdc4-8f20016b7732.png)

![image](https://user-images.githubusercontent.com/30992273/179682894-3c9432d7-07f3-40e8-aaea-37f6edfec9ce.png)
Database execution should not be less than 0ms， I want to know how to collect the time-consuming database

tks！！！

### What you expected to happen

I want to know how to collect the time-consuming database

### How to reproduce

database: pg
oap和ui：9.1.0
agent：8.11.0

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
I don't know can or can't. `0ms` just mean less than `1ms`.
